06-gitops.md

```diff
@@ -31,12 +31,19 @@ GitOps allows a team to author Kubernetes manifest files, persist them in their
    ```bash
    az aks get-credentials -g rg-bu0001a0008 -n $AKS_CLUSTER_NAME
    ```
+   :Note: At this point two important steps are happening: 
+   1. The `az aks get-credentials` command will be fetch a `kubeconfig` containing references to the AKS cluster you have created earlier.
+   1. To _actually_ use the cluster you will need to authenticate. For that, run any `kubectl` commands which at this stage will prompt you to authenticate against Azure Active Directory. For example, run the following command:
+   ```bash
+   kubectl get nodes
+   ```
+   When prompted to authenticate, make sure to use the user that was created earlier on the Azure Active Directory Integration (e.g.: `bu0001a0008-admin`) for this. Once the authentication happens successfully, some new items will be added to your `kubeconfig` file such as an `access-token` with an expiration period. For more information on how this process works in Kubernetes please refer to https://kubernetes.io/docs/reference/access-authn-authz/authentication/#openid-connect-tokens)
```

Comment:
My only concern here is the "make sure to use the user that was created earlier" call out here.  If you forked this, you could use any user (or group) you assigned to the k8s rbac role that has permissions.  Any chance we can just remove this first sentence, but keep the second two?
```suggestion
   Once the authentication happens successfully, some new items will be added to your `kubeconfig` file such as an `access-token` with an expiration period. For more information on how this process works in Kubernetes please refer to https://kubernetes.io/docs/reference/access-authn-authz/authentication/#openid-connect-tokens)
```

Reply:
For sure. That makes sense to me as well.

Reply:
cool, if you commit that suggestion I'll go ahead and merge.  Thanks!